### PR TITLE
Improve spelling and grammar

### PR DESCRIPTION
**Pull request by Kaktushose**

### CODE REVIEW

File: Discord.js/README.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
 # [Discord.js](https://github.com/discordjs/discord.js) updated  
----
+
 > Playlist: https://www.youtube.com/playlist?list=PLthqW7GiLEwIu-ZaXgw9_oT2WzzmF8TT2
 
-* [Preparation](#Preparation)
+* [Prerequisites](#Preparation)
```
Comment (Ron31) [inline Discord.js/README.md]: Wrong linking

File: Discord.js/README.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
+Make sure you have installed the latest version of [Node.js](https://nodejs.org/en/download/).
 
-#### 1. Create the Project and install all Dependencies
+#### 1. Create the project and install all dependencies
 Open your terminal in the folder you want to store the bot in.
 ``` BASH
-# Check if node is installed
+# Check if Node is installed
 node --version
 # Create a node project and type in whatever you want but make sure your main file is main.js
```
Comment (Ron31) [inline Discord.js/README.md]: Write Node big here

File: Discord.py/README.md [hunk 3]
```diff
@@ -1,32 +1,35 @@
 # [Discord.py](https://github.com/Rapptz/discord.py) updated  
----
+
 > Playlist: https://www.youtube.com/playlist?list=PLthqW7GiLEwL0GrBM6P1BKJbHWgk0-9Kj
 
-* [Preparation](#Preparation)
+* [Prerequisites](#Prerequisites)
```
Comment (Ron31) [inline Discord.py/README.md:5]: Wrong linking

**Review by Ron31 — CHANGES_REQUESTED** [review]

File: Discord.py/README.md (hunk 3 above)
Comment (Kaktushose) [reply Discord.py/README.md:5]: The linking is correct here

**Review by Ron31 — APPROVED** [review]

**Pull request merged**